.github/renovate.json5

```
@@ -51,6 +51,27 @@
       matchUpdateTypes: ['major'],
       enabled: false,
     },
+    {
+      // Disable frequent updates for dev tooling
+      // These can be updated manually but we still want to be notified about new majors
+      matchPackageNames: [
+        'commitlint',
+        'husky',
+        'lint-staged',
+        'linters',
+      ],
```

[matchPackageNames](https://docs.renovatebot.com/configuration-options/#matchpackagenames) expects a list of _exact_ package names, so `commitlint` (not one of our direct dependencies anyhow) and `linters` should not be here.  

```suggestion
      extends: ['packages:linters'],
      matchPackagePrefixes: ['@commitlint'],
      matchPackageNames: [
        'husky',
        'lint-staged'
      ],
```

see [packages:linters](https://docs.renovatebot.com/presets-packages/#packageslinters) and [matchPackaagePrefixes](https://docs.renovatebot.com/configuration-options/#matchpackageprefixes)